tests/zfs-tests/tests/functional/no_space/enospc_rm.ksh

```diff
@@ -51,11 +52,20 @@ log_must zfs create $TESTPOOL/$TESTFS
 log_must zfs set mountpoint=$TESTDIR $TESTPOOL/$TESTFS
 log_must zfs set compression=off $TESTPOOL/$TESTFS
 
-log_note "Writing files until ENOSPC."
+log_note "Writing Big(1G) files until ENOSPC."
 log_mustnot_expect "No space left on device" fio --name=test \
     --fallocate=none --rw=write --bs=1M --size=1G --numjobs=4 \
     --sync=1 --directory=$TESTDIR/ --group_reporting
 
+log_must rm $TESTDIR/test.*
+log_must test -z "$(ls -A $TESTDIR)"
+log_must sleep 30
```

Comment:
Is the delay here really needed?  I fear it's going to result in an unreliable test case.  Perhaps we could call `sync_pool $TESTPOOL` here a few times to force the blocks to be reclaimed more quickly?

Reply:
sync_pool $TESTPOOL is appropriate, thanks.